nutils/function.py

```diff
@@ -2629,11 +2629,11 @@ def lower(self, **kwargs: Any) -> evaluable.Array:
 
   @util.cached_property
   def _computed_support(self) -> Tuple[types.frozenarray, ...]:
-    support = [set() for i in range(self.ndofs)] # type: List[Set[int]]
+    support = [[] for i in range(self.ndofs)] # type: List[List[int]]
     for ielem in range(self.nelems):
       for dof in self.get_dofs(ielem):
-        support[dof].add(ielem)
-    return tuple(types.frozenarray(numpy.fromiter(sorted(ielems), dtype=int), copy=False) for ielems in support)
```

Comment:
I believe the new implementation does not guarantee uniqueness because `self.get_dofs` is not unique under rare conditions:

```python
from nutils import mesh
topo, geom = mesh.rectilinear([2], periodic=[0])
basis = topo.basis('spline', degree=2)
basis.get_dofs(0) # [0, 1, 0]
```

We use `_computed_support` as a fallback in `get_support`, which does guarantee uniqueness. Either we should lift this restriction or make sure `_computed_support` returns unique arrays. I'm in favor of the latter.

Reply:
done.